shoop/admin/modules/orders/static_src/create/view/index.js

```diff
@@ -32,35 +32,45 @@ export default function view() {
                 m("button.btn.btn-success.btn-lg.pull-right" + (creating ? ".disabled" : ""), {
                     disabled: creating,
                     onclick: () => {
-                        store.dispatch(beginCreatingOrder());
+                        store.dispatch(beginFinalizingOrder());
                     }
                 }, m("i.fa.fa-check"), " " + gettext("Confirm"))
             ])
         );
     } else {
-        return m("div.container-fluid",
-            (choices.length > 1 ? contentBlock("i.fa.fa-building", gettext("Select Shop"), shopSelectView(store)) : null),
-            contentBlock("i.fa.fa-cubes", gettext("Order Contents"), orderLinesView(store, creating)),
-            contentBlock("i.fa.fa-user", gettext("Customer Details"), customerSelectView(store)),
-            contentBlock("i.fa.fa-truck", gettext("Shipping Method"), shipmentMethodSelectView(store)),
-            contentBlock("i.fa.fa-credit-card", gettext("Payment Method"), paymentMethodSelectView(store)),
-            m("div.order-footer",
-                m("div.text", m(
-                    "small",
-                    gettext("Method rules, taxes and possible extra discounts are calculated after proceeding."))
-                ),
-                m("div.text", m("h2", m("small", gettext("Total") + ": "), total + " " + selected.currency)),
-                m("div.proceed-button", [
-                    m("button.btn.btn-success.btn-block" + (creating ? ".disabled" : ""), {
-                        disabled: creating,
-                        onclick: () => {
-                            if(!source) {
-                                store.dispatch(retrieveOrderSourceData());
+        return [
+            m("div.container-fluid",
+                m("button.btn.btn-gray.btn-inverse.pull-right", {
+                    onclick: () => {
+                        window.localStorage.setItem("resetSavedOrder", "true");
+                        window.location.reload();
+                    }
+                }, m("i.fa.fa-undo", " " + gettext("Discard Changes")))
```

Comment:
Shouldn't it be `m("i.fa.fa-undo"), " " + gettext("Discard Changes")`, i.e. the text shouldn't be inside the `i` element. Otherwise it will affect styling.